react/features/base/tracks/actions.js

```diff
@@ -188,12 +188,19 @@ export function createLocalTracksA(options = {}) {
 }
 
 /**
- * Calls JitsiLocalTrack#dispose() on all local tracks ignoring errors when
+ * Calls JitsiLocalTrack#dispose() on the given track or on all local tracks (if none are passed) ignoring errors if
  * track is already disposed. After that signals tracks to be removed.
  *
+ * @param {JitsiLocalTrack|null} [track] - The local track that needs to be destroyed.
  * @returns {Function}
  */
-export function destroyLocalTracks() {
+export function destroyLocalTracks(track = null) {
+    if (track) {
```

Comment:
this could be a dedicated method if on optional parameter it executes completely different body

Reply:
I don't think there is much of a difference there. If a track object is passed, only that track is disposed and removed from conference. All the local tracks are removed and disposed otherwise.

Reply:
but the method is called destroyLocalTrack**s**, what you're doing here is destroying a single track, then why not create a separate method called `destroyLocalTrack` which always requires the `track` value to be there?

Reply:
it's just that this optional argument does not make sense and complicated semantics of this method without any added benefit